docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+
+services:
+    application:
+      build:
+          dockerfile: ./Dockerfile
+          context: .
+      command: yarn watch
+      environment:
+          RAZOR_API_KEY: pleasechangeme
```

Comment:
Maybe we can use the .env for the razor key :) 

https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option